src/itype.cpp

```diff
@@ -376,3 +376,13 @@ std::tuple<encumbrance_modifier_type, int> armor_portion_data::convert_descripto
 }
 
 std::map<itype_id, std::set<itype_id>> islot_magazine::compatible_guns;
+
+std::set<itype_id> known_bad_density::known_bad;
+
+void known_bad_density::load( const JsonObject &jo )
+{
+    jo.allow_omitted_members();
```

Comment:
I don't think you need this.
```suggestion
```

Reply:
Yeah, probably. I just copied it from the charge removal/migration blacklists.